SimpleWAWS/Code/CsmExtensions/CsmSubscriptionExtensions.cs

```diff
@@ -22,18 +46,19 @@ public static partial class CsmManager
                 if (deleteBadResourceGroups)
                 {
                     var deleteBadResourceGroupsTasks = csmResourceGroups.value
-                        .Where(r => r.tags != null
-                                    && ((r.tags.ContainsKey("Bad") || 
-                                    (subscription.Type==SubscriptionType.AppService?!r.tags.ContainsKey("FunctionsContainerDeployed"): !r.tags.ContainsKey(Constants.SubscriptionType)))
-                                    && (!r.tags.ContainsKey("UserId"))
+                        //Some orphaned resourcegroups can have no tags. Okay to clean once in a while since they dont have any sites either
+                        .Where(r => ((r.tags == null && IsSimpleWawsResourceName(r)) 
```

Comment:
are these just formatting changes? if they are, can you revert them? I can't tell from the diff if anything actually changed.